### PR TITLE
chores: fix some fmt err

### PR DESCRIPTION
**Pull request by ls-2018**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Comment by hormes** [comment]
Great job.

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: pkg/koordlet/runtimehooks/hooks/coresched/helper.go [hunk 1]
```diff
@@ -126,7 +126,7 @@ func (p *Plugin) addCookie(pids []uint32, groupID string) (uint64, []uint32, err
 // TODO: refactor to resource updater.
 func (p *Plugin) assignCookie(pids, siblingPIDs []uint32, groupID string, targetCookieID uint64) ([]uint32, []uint32, error) {
 	if len(pids) <= 0 {
-		klog.V(6).Infof("aborted to assign PIDs cookie for group %s, target cookie %v, no PID",
+		klog.V(6).Infof("aborted to assign PIDs cookie for group %d, target cookie %v, no PID",
```
Comment (saintube) [inline pkg/koordlet/runtimehooks/hooks/coresched/helper.go:129]: need to correct the args' order

**Review by saintube — APPROVED** [review]
/lgtm

**Pull request merged**